release notes/v0.36.0.md

```diff
@@ -0,0 +1,136 @@
+
+ERRO[0000] line 6
+        at f2 (file:///some/path/imported.js:6:2(2))
+        at file:///some/path/loadtest.js:4:2(4)
+        at native  executor=per-vu-iterations scenario=default source=stacktrace
+```
+
+### Ability to abort tests ([#2093](https://github.com/grafana/k6/pull/2093))
+
+Thanks to the contribution of [@gernest](https://github.com/gernest), k6 now has the ability to abort a test. [#2093](https://github.com/grafana/k6/pull/2093) made the `k6/execution` expose the `test.abort()` function, which allows to immediately abort a k6 load test run with the exit code `108`. Note that aborting a test is also fully supported in k6 cloud, preserving the same user-experience for both local and cloud users. 
+
+Aborting is possible during initialization:
+```javascript
+import exec from "k6/execution";
+exec.test.abort();
+```
+
+As well as inside the default function:
+```javascript
+import exec from "k6/execution";
+
+export default function() {
+  // Note that you can abort with a specific message too
+  exec.test.abort("this is the reason");
+}
+
+export function teardown() {
+  console.log("teardown will still be called after test.abort()");
+}
+```
+
+Or during the VU initialization:
+```javascript
+import exec from "k6/execution";
+
+// This won't fail on initial parsing of the script, but on VU initialization.
+if (__VU == 1) {
+  exec.test.abort();
+}
+
+export default function() {}
+```
+
+### k6 inspect extended output ([#2279](https://github.com/grafana/k6/pull/2279))
+
+Following [#2279](https://github.com/grafana/k6/pull/2279), the `k6 inspect` command now supports an `--execution-requirements` flag. When used, the command's output will include fields related to the execution requirements, by deriving k6's configuration from the execution context, and including  the `maxVUs` and `totalDuration` fields to the output.
+
+
+### Forcing HTTP/1 protocol ([#2222](https://github.com/grafana/k6/pull/2222))
+
+Thanks to the work of [@sjordhani22](https://github.com/sjordhani22), [#2222](https://github.com/grafana/k6/pull/2222) made it possible to force k6 to use version 1.1 of the protocol when firing HTTP requests. 
+
+It can be done by setting the `http2client=0` value in the `GODEBUG` environment variable:
+
+```
+GODEBUG=http2client=0 k6 run testscript.js
+```
+
+**Nota Bene**: the usage of the `GODEBUG` variable is considered temporary, and expected to change in the future. If you start using this feature, keep an eye out for potential future changes.
+
+## Breaking changes
+
+### Restricting file opening to init context
+
+Opening files is now restricted to the init context. While it was somewhat possible, yet [unreliably](https://github.com/grafana/k6/issues/1771), to open files during other stages of the test in the past: [#2314](https://github.com/grafana/k6/pull/2314) ensures that k6 would now throw an error in a similar scenario.
+
+Statements such as the following, will now throw an error.
```

Comment:
unless `arr.json` wasn't opened somewhere else, see [old workarounds](https://k6.io/docs/examples/data-parameterization/#old-workarounds) in the data parameterization documentation

Reply:
I have rephrased, let me know if that works for you. Otherwise, please propose an alternative, as I'm not totally sure I get the fine context and details of this yet 🐻